apps/chat/models.py

```diff
@@ -64,9 +77,18 @@ def created_at_datetime(self):
         return quote(self.created_at.isoformat())
 
     def to_langchain_dict(self) -> dict:
+        return self._get_langchain_dict(self.content, self.message_type)
+
+    def summary_to_langchain_dict(self) -> dict:
+        return self._get_langchain_dict(self.summary, ChatMessageType.SYSTEM)
```

Comment:
since this is calling the `_get_langchain_dict`, is there any reason to keep `_get_langchain_dict` around?